ports/ngtcp2/portfile.cmake

```
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ngtcp2/ngtcp2
     REF "v${VERSION}"
-    SHA512 3d266b80fbc44bf0711f4791db4787d80c989beb84ff4fe33e0f8fd25b8bb6d6d768bcbf990ca4bde0bcbaad5c7566bfb6a431b5bbec844247d8a4b417d451d9
+    SHA512 8be12732e27eaabfbedae83ceee66ae0a269edebc614a5dd7645b8019d1f394be91391943baade93bad71e6100e22e46240b2b01c97ba3dedd7ef343113351e1
     HEAD_REF master
```

```suggestion
    HEAD_REF main
```
The default branch of `ngtcp2` is main.